<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>980</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1064</width>
    <height>980</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1064</width>
    <height>980</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>virtual Packaging Machine</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/logo A&amp;R.svg</normaloff>images/logo A&amp;R.svg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QFrame" name="frmControlInterface">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frmButtons">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnStart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStop">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>75</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEmergency">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>75</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>EMERGENCIA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReset">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>75</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frmDisplays">
     <property name="geometry">
      <rect>
       <x>249</x>
       <y>9</y>
       <width>181</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>52</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vloLbldisplay">
       <item>
        <widget class="QLabel" name="lblSpeed">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>SPEED:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblActPos">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ACT POS:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblIpos">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>IPOS:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>66</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vloLcdDisplay">
       <item>
        <widget class="QLCDNumber" name="lcdSpeed"/>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdActPos"/>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdIpos"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QFrame" name="frmMainSwitch">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>0</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vloMainSwitch"/>
     </widget>
     <widget class="QLabel" name="lblMainSwitch">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>129</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>INTERRUPTOR GENERAL</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lblPLCcom">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CONEXIÃ“N OFFLINE</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmMachineInterface">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>661</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frmConveyor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>450</width>
       <height>150</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblImgConveyor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>25</y>
        <width>400</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/logo A&amp;R.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>box1</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box2</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box3</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box4</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box5</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box6</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box7</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box8</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox9">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box9</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox10">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box10</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox11">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box11</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBox12">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>box12</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnTest">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>452</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>test</string>
     </property>
    </widget>
    <widget class="QFrame" name="frmFeeder1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>19</y>
       <width>80</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frmFeeder2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>81</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>81</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vloFeeder2"/>
     </widget>
    </widget>
    <widget class="QFrame" name="frmFeeder3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>81</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>81</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vloFeeder3"/>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>81</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vloFeeder1"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>29</y>
       <width>191</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutTEST"/>
    </widget>
   </widget>
   <widget class="QFrame" name="frmSettingsInterface">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>271</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frmInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>940</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Modelo Virtual VS7WB_v1</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSuscribe">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>101</width>
       <height>75</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/logo A&amp;R.svg</normaloff>images/logo A&amp;R.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
